client/galaxy/scripts/mvc/ui/ui-select-content.js

```diff
@@ -7,16 +7,17 @@ import Ui from "mvc/ui/ui-misc";
 import Select from "mvc/ui/ui-select-default";
 import { getGalaxyInstance } from "app";
 
-/** Batch mode variations */
-var Batch = { DISABLED: "disabled", ENABLED: "enabled", LINKED: "linked" };
+/** Batch mode letiations */
+let Batch = { DISABLED: "disabled", ENABLED: "enabled", LINKED: "linked" };
```

Comment:
We should prefer `const` for situations like this where there is no rebinding of the variables.

Reply:
Thanks @dannon.

Reply:
@guerler Sure, no worries.  I just opened https://github.com/galaxyproject/galaxy/pull/7860 that'll fix the existing occurrences of this and will lint it moving forward, too.